### ISSUE TITLE
No mail for published new law

### ISSUE THREAD

**Issue by willouuu** [DemocracyOS/democracyos#1443]
Hello,

Can you help us with this problem:
There is no message sent to the owner of a forum when a new law is published even if the parameter is set to 'Yes'.
Also, we never see notifications in the notifications web page.
On the other hand, a message is sent in response to an argument (Someone replied to your argument).
Thanks in advance,

Best regards.

**Comment by franciclo** [comment]
Hi, make sure you are using the latest version (now 2.8.2), we make some modification to the notifier module lately.

**Comment by willouuu** [comment]
Yes of course, we have a latest version to github repo.
At the same time, we wonder whether we should use "make" or "docker-compose" to run the application because the behavior is different for notifications? What do you recommend?

**Comment by franciclo** [comment]
Oh, sorry, that option shouldn't be displayed in multiForum mode.
We have to fix that, sending notifications to all participants of the forum.
I will close this when its done.
We recomend using docker, but it shouldn't behave different than make in any aspect of the functionality.

**Comment by willouuu** [comment]
Okay
Thank you

We also have this error log for new law but it is still created:

```
  democracyos:l10n Setting locale: en +0ms
  democracyos:api GET /api/v2/topics?random=0.770900085057463 +0ms
  democracyos:api Error: GET /api/v2/topics?random=0.770900085057463 { Error: Invalid request parameters.
    at validationErrorHandler (/local/DemocracyOS/democracyos-master.bak/lib/api-v2/boot/index.js:40:17)
    at Layer.handle_error (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:315:13)
    at /local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:335:12)
    at next (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:275:10)
    at mongooseCastErrorHandler (/local/DemocracyOS/democracyos-master.bak/lib/api-v2/boot/index.js:25:12)
    at Layer.handle_error (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/layer.js:71:5)
    at trim_prefix (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:315:13)
    at /local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:335:12)
    at next (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:275:10)
    at Layer.handle_error (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/layer.js:67:12)
    at trim_prefix (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:315:13)
    at /local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:335:12)
    at next (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:275:10)
    at Layer.handle_error (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/layer.js:67:12)
    at trim_prefix (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:315:13)
    at /local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:284:7
    at Function.process_params (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:335:12)
    at next (/local/DemocracyOS/democracyos-master.bak/node_modules/express/lib/router/index.js:275:10)
  status: 400,
  code: 'INVALID_REQUEST_PARAMS',
  info: { query: [ [Object] ] } } +6ms
  democracyos:jwt:token Token decoded successfully +3ms 
```

Do you want a new ticket?

**Comment by franciclo** [comment]
That error is fixed on 2.8.3

**Comment by willouuu** [comment]
Good
Thanks
